src/routes/news/[...slug].svelte

```diff
@@ -0,0 +1,92 @@
+<script context="module">
+    import newsTargets from 'news-targets';
+    export async function preload({ params, query }) {
+        // the `slug` parameter is available because
+        // slug is meant to be the part of a URL which identifies a particular page on a website.
+        const [slug, title] = params.slug;
+        const res = await this.fetch(`news/${slug}.json`);
+        const data = await res.json();
+        // This if-else statement is used to make sure that 
+        // the web page can be displayed (redirect status of 200). 
+        // Otherwise, an error message will be displayed.
+        if (res.status === 200) {
+            return {
+                news,
+                newsIndex: data.newsIndex,
+            };
```

Comment:
I think this is not supposed to be here since `news` is defined on line 23.
![image](https://user-images.githubusercontent.com/39835686/105886612-91cbe200-6045-11eb-801c-322b86099d48.png)

What I was suggesting before is that.
We simply moved the error-handling to this place.

So, we can have like
```
if (res.status != 200) {
    this.error(res.status, data.message);
    return; // we stop the function immediately
}
```

Later on after line 27, we just need to return the objects without any other if-else error checking since we have handled it above.

```
return {
    news,
    newsIndex: data.newsIndex,
};
```